Lombiq.NodeJs.Extensions/package.json

```diff
@@ -24,35 +26,35 @@
     "sass": "^1.52.3",
     "stylelint": "^14.9.1",
     "stylelint-config-standard-scss": "^4.0.0",
-    "terser": "^5.14.1"
+    "terser": "^5.14.1",
+    "textlint": "^12.2.1",
+    "textlint-rule-common-misspellings": "^1.0.1",
+    "textlint-rule-doubled-spaces": "^1.0.0",
+    "textlint-rule-no-todo": "^2.0.1",
+    "textlint-rule-no-zero-width-spaces": "^1.0.1"
   },
   "scripts": {
     "// shell docs": "https://github.com/yarnpkg/berry/blob/master/packages/yarnpkg-shell/tests/shell.test.ts",
     "// advice    ": "When using glob patterns (*/**) as command arguments, wrap the whole argument in quotes.",
     "-": "------------------------------------------------------------------------------------------------------------",
-    "build": "run-p --print-label --continue-on-error \"build:*\"",
-    "compile": "run-p --print-label --continue-on-error \"compile:*\"",
-    "lint": "run-p --print-label --continue-on-error \"lint:*\"",
-    "clean": "run-p --print-label --continue-on-error \"clean:*\"",
-    "watch": "run-p --print-label --continue-on-error \"watch:*\"",
```

Comment:
Why remove `--print-label`?

Reply:
I thought it interfered with the MSBuild warnings, but it turns out something else was. I've restored the labels.